target/linux/rockchip/armv8/base-files/etc/board.d/01_leds

```diff
@@ -28,6 +28,10 @@ friendlyarm,nanopi-r5s)
 	ucidef_set_led_netdev "lan1" "LAN1" "green:lan-1" "eth1"
 	ucidef_set_led_netdev "lan2" "LAN2" "green:lan-2" "eth2"
 	;;
+sinovoip,rk3568-bpi-r2pro)
+	ucidef_set_led_netdev "wan" "WAN" "blue:status" "eth0"
```

Comment:
this is not okay... the LED on-board is not used for network activity.

Reply:
Okay, I removed it